go.mod

```diff
@@ -5,11 +5,12 @@ go 1.15
 require (
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.19.2
-	k8s.io/apimachinery v0.19.2
+	k8s.io/api v0.20.0-beta.0
+	k8s.io/apimachinery v0.20.0-beta.0
 	k8s.io/apiserver v0.19.0
-	k8s.io/client-go v0.19.2
+	k8s.io/client-go v0.20.0-beta.0
 	k8s.io/code-generator v0.19.0
 	k8s.io/component-base v0.19.2
+	k8s.io/component-helpers v0.20.0-alpha.2.0.20201030045251-0fa5e458da80
```

Comment:
Until there's the first release of k8s.io/component-helpers, there are no stability guarantees of the API in the right sense even though the stability intention is strongly enforced in  `k8s.io/component-helpers` PR reviews. Given the descheduler is the first repo outside of kubernetes org to import the new staging repo and for the stability assumption mentioned above, we can use the descheduler repo as a guinea pig until the first release is out.